.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+module.exports = {
```

Comment:
Please remove this file. Linting for these smaller projects is handled via tsdx (which is how the CI lints), so that we don't have to update each repo individually every time we make a change! If we'd like an eslint file in the config I guess we could do it via symbolic link, but that might get problematic with git...

Reply:
Ah, I didn't understand the bigger projects. VSCode wasn't picking up the linting/formatting rules, so I had ESLint generate this file and thought I'd include it, but what you say makes perfect sense.